README.md

```diff
@@ -1,77 +1,64 @@
 ```
 
-The goal of this stack is to enable decentralized data-structures
-which in turn will enable more decentralized applications.
+## Block Layer (Layer 0)
 
-Many of the specifications in this stack are inter-dependent.
+The block layer encompasses all content addressed block formats and specifies how blocks
+are addressed,how they self-describe their codec for encoding/decoding, and how block link
+between each other.
 
-```
-                          IPLD Dependency Graph
-
-┌─────┐                           ┌────────────┐
-│ CID ├───────────┬───────────────> Raw Blocks │
-└─────┘           │               └─┬──────────┘
-          ┌───────v──────────────┐  │
-┌──────┐  │ Links (Conceptually) │  │
-│ Path │  └───────┬──────────────┘  │           ┌─────────────┐
-└──┬───┘          │                 ├───────────> Replication │
-   │              │                 │           └─────────────┘
-┌──v──────────────v─────────────────┴──────┐
-│                  Codecs                  │
-│  ┌─────┐   ┌───────────────────────────┐ │ ┌────────────────────────────┐
-│  │ Git │   │       Data Model v1       │ │ │  Multi-block Collections   │
-│  └─────┘   │ ┌──────────┐ ┌──────────┐ │ │ │ ┌────────────────────────┐ │
-│ ┌────────┐ │ │ dag-json │ │ dag-cbor │ ├─┼─> │    Specialized Data    │ │
-│ │  BTC   │ │ └──────────┘ └──────────┘ │ │ │ │ Structures & Utilities │ │
-│ └────────┘ └───────────────────────────┘ │ │ └────────────────────────┘ │
-│ ┌────────┐ ┌────────┐                    │ └───┬─────────┬──────────────┘
-│ │ Zcash  │ │ dag-pb │                    │     │         │
-│ └────────┘ └────────┘                    │     │         │
-└──────────────────────────────────────────┘     │         │
-                                                 │         │
-               ┌──────────────────────┐          │         │
-               │ File System (UnixFS) <──────────┘         │
-               └──────────────────────┘                    │
-               ┌───────────────────────┐                   │
-               │ End-User Applications <───────────────────┘
-               └───────────────────────┘
+This layer alone is enough to accomplish basic graph replication across a variety of formats (eth, bitcoin, git, etc).
 
-```
+This layer does not define data structures or types, although many codecs may convert these formats into native types,
+there are no type requirements or assurances about types at the block layer.
+
+## Data Model Layer (Layer 1)
+
+The Data Model Layer describes a set of base required types to be implemented by a subset of IPLD codecs.
+
+With these basic types authors can create various single-block data structures which can be read with predictable
+paths and selectors.
+
+With just the data model layer, several data structures can be authored and put into a single block. These data 
+structures can also link to one another, but a *single* data structures's namespace cannot be spread across many
```

Comment:
I think "data structure namespace" needs a definition.

Reply:
ya, i’m struggling with the language here because we’ve actually just over-loaded the term “data structure.”

like, we *do* have multi-block data structures we just don’t have multi-block **collections**. maybe I just need to replace some of the data structure verbiage with collections and link appropriately.

Reply:
`s/structures's/structures'/`